src/doc/components/Example.js

```diff
@@ -27,14 +31,15 @@ const useStyles = makeStyles({
     paddingTop: 5,
     paddingRight: 10,
   },
-});
+}));
 
 const Example = () => {
   const { editButton, paper, htmlContainer, fileName } = useStyles();
   const { exampleKey } = useParams((params) => params);
   const example = EXAMPLES.find((e) => e.key === exampleKey);
   const [html, setHtml] = useState();
   const [js, setJs] = useState();
+  const mapContainerRef = useRef(null);
```

Comment:
Please don't use refs if they're not necessary. Here you can use conditional styling instead.
```
const [isNavigable, setIsNavigable] = useState(false);
// ...
<div
  style={{ pointerEvents: isNavigable ? 'auto' : 'none' }}
  onClick={() => setIsNavigable (true)}
/>
```

Reply:
I'm sorry if I can't see the obvious, but how do you imagine setting the initial value of isNavigable to true when on Desktop? Because this approach makes theme.breakpoints.down('xs') useless and we would then still need to access media-query to load isNavigable correctly on mount. Or do you want the map interactions to work only on click, regardless of media-query?

Reply:
Alternative in #73